src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -125,6 +176,27 @@ class DataExplorerVisualizations extends React.Component {
       : null;
     const selectedTableRowsCount = this.props.selectedTableRows.length;
     const dropdownConfigs = calculateDropdownButtonConfigs(this.props.explorerTableConfig);
+    const toaster =
```

Comment:
instead of defining this here and reusing the css from `MapDataModel.jsx` it would be good to abstract it out into its own component